src/amici.cpp

```diff
@@ -1623,6 +1997,55 @@ int workBackwardProblem(UserData *udata, TempData *tdata, ReturnData *rdata, con
     return AMICI_SUCCESS;
 }
 
+int workSteadyStateProblem(UserData *udata, TempData *tdata, ReturnData *rdata, void *ami_mem, int it) {
+    /*
+     * tries to determine the steady state of the ODE system by a Newton solver
+     uses forward intergration, if the Newton solver fails
+     *
+     * @param[in] udata pointer to the user data struct @type UserData
+     * @param[in] tdata pointer to the temporary data struct @type UserData
+     * @param[out] tdata pointer to the temporary data struct @type TempData
+     * @param[out] rdata pointer to the return data struct @type ReturnData
+     */
+    
+    int status = (int) *rdata->status;
+    double run_time;
+    clock_t starttime;
```

Comment:
It is used now, but I just noticed:
>             run_time = (double)((clock() - starttime) * 1000) / CLOCKS_PER_SEC;
This does not give you seconds, but 1e3 seconds.